docs/getting-started/golden-path.md

```diff
@@ -15,18 +21,6 @@ who are interested.
 Each document in the path will lead to the next so have a look at the bottom of the page
 for the 'Continue to ...' link to keep going.
 
-Continue to download and install...  
-***WINDOWS***
-
-* [Paymetheus](/getting-started/install-guide#windows-installer)
-* [Command Line](/getting-started/install-guide)
-
-***MAC***
-
-* [Decrediton](https://github.com/decred/decrediton)
-* [Command Line](/getting-started/install-guide#)
-
-***LINUX***
+---
```

Comment:
Thought it might flow better if we gave them a brief description of the applications and explained compatibility which is why I added the operating-systems.md page. As it currently exists, it feels awkward - a beginner has to make a slightly uninformed decision on which path they want to embark on (GUI or CLI). I say slightly because they might correctly assume that Paymetheus is not a Command Line application. Not against putting it back to go directly to the Install guide, or maybe introducing the applications within golden-path.md.